cargo

```diff
@@ -3,6 +3,11 @@
 # shellcheck source=ci/rust-version.sh
 here=$(dirname "$0")
 
+export CI_COMMIT=$(git rev-parse --short HEAD)
```

Comment:
I'd really like to avoid putting other things into `./cargo`.  The purpose of `./cargo` is to select the correct `cargo`, that's it!